medusa/search/__init__.py

```diff
@@ -3,7 +3,9 @@
 """Search module for all Medusa searches."""
 from __future__ import unicode_literals
 
-from enum import Enum
+import sys
+if sys.version_info < (3, 4):
+    from ext2.enum import Enum
```

Comment:
You never import enum in Py >= 3.4

Reply:
You should only add ext2 in Python < 3.  Also Python 3 < 3.4 shouldn't be supported.  Finally your version never imports under Python > 3.4.

Reply:
Good point

Reply:
Also consider using IntEnum since they were traditionally ints, although im not sure if the backport supports int.  You could create your own IntEnum if it doesnt.  `class IntEnum(builtins.int, Enum)`.